src/test/cypress/integration/exam/ExamAssessment.spec.ts

```diff
@@ -0,0 +1,184 @@
+        examTitle = 'exam' + generateUUID();
+        cy.login(admin);
+    });
+
+    afterEach('Delete Exam', () => {
+        cy.login(admin);
+        courseManagementRequests.deleteExam(exam);
+    });
+
+    after('Delete Course', () => {
+        cy.login(admin);
+        courseManagementRequests.deleteCourse(course.id);
+    });
+
+    describe('Exam exercise Assessment', () => {
+        beforeEach('Create Exam', () => {
+            prepareExam(dayjs().add(30, 'seconds'));
+        });
+
+        it('Assess a modeling exercise submission', () => {
+            courseManagementRequests.createModelingExercise({ exerciseGroup }).then((modelingResponse) => {
+                courseManagementRequests.generateMissingIndividualExams(exam);
+                courseManagementRequests.prepareExerciseStartForExam(exam);
+                cy.login(student, '/courses/' + course.id + '/exams/' + exam.id);
+                examStartEnd.startExam();
+                cy.contains(modelingResponse.body.title).should('be.visible').click();
+                modelingEditor.addComponentToModel(1);
+                modelingEditor.addComponentToModel(2);
+                modelingEditor.addComponentToModel(3);
+                cy.intercept(PUT, BASE_API + 'exercises/*/modeling-submissions').as('createModelingSubmission');
+                cy.contains('Save').click();
+                cy.wait('@createModelingSubmission');
+                cy.get('#exam-navigation-bar').find('.btn-danger').click();
+                examStartEnd.finishExam();
+                cy.login(tutor, '/course-management/' + course.id + '/exams');
+                cy.contains('Assessment Dashboard', { timeout: 60000 }).click();
+                assessmentDashboard.startAssessing();
+                modelingAssessment.addNewFeedback(2, 'Noice');
+                modelingAssessment.openAssessmentForComponent(1);
+                modelingAssessment.assessComponent(1, 'Good');
+                modelingAssessment.openAssessmentForComponent(2);
+                modelingAssessment.assessComponent(0, 'Neutral');
+                modelingAssessment.openAssessmentForComponent(3);
+                modelingAssessment.assessComponent(-1, 'Wrong');
+                assessmentDashboard.submitModelingAssessment().then((assessmentResponse) => {
+                    expect(assessmentResponse.response?.statusCode).to.equal(200);
+                });
+                cy.login(student, '/courses/' + course.id + '/exams/' + exam.id);
+                cy.contains('2 of 10 points').should('be.visible');
+            });
+        });
+
+        it('Assess a text exercise submission', () => {
+            const exerciseTitle = 'Cypress Text Exercise';
+            courseManagementRequests.createTextExercise({ exerciseGroup }, exerciseTitle);
+            courseManagementRequests.generateMissingIndividualExams(exam);
+            courseManagementRequests.prepareExerciseStartForExam(exam);
```

Comment:
Same as above. This should be in a before hook

Reply:
:+1: